### DISCUSSION TITLE
Could not activate pairing mode

### DISCUSSION THREAD

**Discussion by denstu** [Koenkk/zigbee2mqtt#18665] [Devices]
Hello everybody, 

I have this device
https://amzn.eu/d/ekC5aQS

It worked perfekt. Now i changed the coordinator and have to repair All devices. But i cant put that one into pairing mode. 

Switch on and off 5 Times, doesnt work. Tried it several Times, 4x, 6x... No response from the bulb. 

Can anyone help? Or is it broken? 

Thanks in advance.

**Comment by francisp2** [comment]
Tried touchlink ?

https://www.zigbee2mqtt.io/guide/usage/touchlink.html

**Reply by denstu** [reply]
Thanks for this input. I did not know about touchlink, but it worked perfectly. Thanks!
